site/gatsby-site/src/components/discover/OptionsModal.js

```diff
@@ -56,6 +57,17 @@ function OptionsModal({ searchState, setSearchState }) {
             <Form.Label for="hide-duplicates-modal">
               <Trans>1st report only</Trans>
             </Form.Label>
+
+            <ToggleDisplayIssues attribute="is_incident_report" id="display-issues" />
+            <Form.Label htmlFor="display-issues">
+              <Trans>Display Issues</Trans>
+            </Form.Label>
+
+            <ToggleOnlyIssues attribute="is_incident_report" id="only-issues" />
+            <Form.Label htmlFor="only-issues">
+              <Trans>Only Issues</Trans>
```

Comment:
Missing translations for these two filters. Also the UI is broken on mobile:

![image](https://user-images.githubusercontent.com/6564809/199126577-a09dc762-3373-44c5-b392-90e714e85f3c.png)